logging/src/main/kotlin/com/ritense/logging/LoggingAutoConfiguration.kt

```diff
@@ -56,6 +58,23 @@ class LoggingAutoConfiguration {
         return LoggableResourceAspect()
     }
 
+    @Bean
+    @ConditionalOnMissingBean(LoggingEventDeletionService::class)
+    fun loggingEventDeletionService(
+        @Value("\${valtimo.logging.retentionInMinutes:10080}") retentionInMinutes: Long,
```

Comment:
I think this might be too low for a default value. Maybe we should go for a month?

Reply:
Changed